### PR TITLE
skip dotnet first experience + run CI on macOS

### PR DESCRIPTION
**Pull request by filipw**
so far we only ran on Windows + Linux.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,24 +4,16 @@ sudo: required
 dist: trusty
 env:
   global:
-    secure: m2PtYwYOhaK0uFMZ19ZxApZwWZeAIq1dS//jx/5I3txpIWD+TfycQMAWYxycFJ/GJkeVF29P4Zz1uyS2XKKjPJpp2Pds98FNQyDv3OftpLAVa0drsjfhurVlBmSdrV7GH6ncKfvhd+h7KVK5vbZc+NeR4dH7eNvN/jraS//AMJg=
+    - secure: m2PtYwYOhaK0uFMZ19ZxApZwWZeAIq1dS//jx/5I3txpIWD+TfycQMAWYxycFJ/GJkeVF29P4Zz1uyS2XKKjPJpp2Pds98FNQyDv3OftpLAVa0drsjfhurVlBmSdrV7GH6ncKfvhd+h7KVK5vbZc+NeR4dH7eNvN/jraS//AMJg=
+    - DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true
```
Comment (khellang) [inline .travis.yml:8]: Should we also include `DOTNET_CLI_TELEMETRY_OPTOUT: 1`?

https://github.com/aspnet/Universe/blob/18575e8879fd828d22230b17b0949010fe3cf06b/.travis.yml#L6-L7

**Review by khellang — APPROVED** [review]
LGTM!

File: .travis.yml (hunk 1 above)
Comment (filipw) [reply .travis.yml:8]: good idea

**Comment by khellang** [comment]
👏

**Pull request merged**